WooCommerce/Classes/Authentication/Navigation Exceptions/JetpackErrorViewModel.swift

```diff
@@ -49,8 +49,10 @@ struct JetpackErrorViewModel: ULErrorViewModel {
     }
 
     func didTapSecondaryButton(in viewController: UIViewController?) {
-        let refreshCommand = NavigateToRoot()
-        refreshCommand.execute(from: viewController)
+        // To be removed when PR #3186 is merged.
```

Comment:
This should not be part of the diff, it is temporary, and I will remove it before merging. 

But, the API of WPAuthenticator is a bit ahead